tests/cases/crashes/layout_deleted_item.slint

```diff
@@ -64,9 +64,9 @@ assert!(!instance.get_hover());
 
 ```js
 var instance = new slint.TestCase({
-    clicked: function() { var x = instance.model; x.pop(); instance.model = x; }
+    clicked: function() { var x = instance.model; x.remove(x.length - 1, 1); instance.model = x; }
 });
-instance.model = [1, 2];
+instance.model = new slintlib.ArrayModel([1, 2]);
```

Comment:
Can I not assign array to model property anymore?

Reply:
Simon and I talked about that topic. We agreed that it make sense to make it only possible to set and get a model instead of an pure array. What you set and what you get for a model should behave like a model with all it's features and also with the complete notify mechanism. Array has e.g. a different api.

Reply:
More specifically: setter accepts model or array. But getter always returns a model.

Since the whole thing is labeled beta, I thought that would be acceptable.

I think the getter sometimes returning one thing and sometimes another is worse.